### PR TITLE
retroshare: Update to v0.6.7.2

### PR DESCRIPTION
**Pull request by ermo**
**Summary**
- Update OBS submodule
- Fix build failure (resolves getsolus/packages#2884)

**Test Plan**

- Open retroshare, create entropy, add username and password, start up, check that interface looks functional.

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

**Comment by ermo** [comment]
I removed a specific patch from this build. So I need a frequent user of retroshare to actually run through this build and check that it functions like they're used to.

File: packages/r/retroshare/monitoring.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+releases:
+  id: ~ # No id on https://release-monitoring.org as of 2024-11-01
```
Comment (EbonJaeger) [inline packages/r/retroshare/monitoring.yml:2]: The project has been added [here](https://release-monitoring.org/project/375304/) :tada:

File: packages/r/retroshare/package.yml [hunk 2]
```diff
@@ -24,8 +25,17 @@ builddeps  :
     - pkgconfig(sqlcipher)
     - pkgconfig(x11)
     - pkgconfig(xscrnsaver)
+environment: |
+    export CC=clang
+    export CXX=clang++
```
Comment (EbonJaeger) [inline packages/r/retroshare/package.yml:30]: Will it work if we do `clang: yes` here instead of exporting the environment variables?

**Comment by malfisya** [comment]
Do we still need the frequent user? It seems rare. It seems working fine. I want to get this over the line soon, here is a few options :
Option 1: We can get this to unstable after the sync, so unstable user can test it.
Option 2: YOLO sync it to stable this week and wait until an actual user complain about it. We can add the patch later
Option 3: We can test it among ourselves by exchanging retroshare ID. If it is good, sync it. If it is not, add the patch and test it again.

**Review by malfisya — APPROVED** [review]

**Pull request merged**